src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.InlineRenameLocationSet.cs

```diff
@@ -17,14 +18,19 @@ internal abstract partial class AbstractEditorInlineRenameService
         private class InlineRenameLocationSet : IInlineRenameLocationSet
         {
             private readonly LightweightRenameLocations _renameLocationSet;
+            private readonly CodeCleanupOptionsProvider _fallbackOptions;
             private readonly SymbolInlineRenameInfo _renameInfo;
 
             public IList<InlineRenameLocation> Locations { get; }
 
-            public InlineRenameLocationSet(SymbolInlineRenameInfo renameInfo, LightweightRenameLocations renameLocationSet)
+            public InlineRenameLocationSet(
+                SymbolInlineRenameInfo renameInfo,
+                LightweightRenameLocations renameLocationSet,
+                CodeCleanupOptionsProvider fallbackOptions)
```

Comment:
this data is now captured and passed along, simplifying its acquisition later on.